Template for setting up eventbrite (#11)
RSE-Sheffield/conda-environments-for-effective-and-reproducible-research: .github/setup-eventbrite.md

about: Instructions for setting up materials ready for next session. To be completed by lead instructor
title: Scheduling courses with Eventbrite on [DATE-OR-SESSION-DESCRIPTION] for [INSTRUCTOR-NAME]
labels: session-setup
assignees: ''

---

The lead instructor needs to schedule events on [Eventbrite](https://www.eventbrite.com) using the [RSE
Sheffield](https://www.eventbrite.co.uk/o/rse-sheffield-29660305889) organisation. If you have not already been setup as
an administrator for this you should request to be included by another team member (see [RSEs
Handbook](https://github.com/RSE-Sheffield/rses-handbook/blob/master/pages/eventbrite.Rmd) for further details).

**NB** - The easiest approach is to copy an existing event. From the page listing events select **Past** and click on
the three dots to the right of a past event and select `Copy Event`. Edit the title to remove `Copy of` and update the
dates and times.

## Basic Info

The following fields should be entered into each field on the **Basic Info** page.

| Field               | Value                                                     |
|:--------------------|-----------------------------------------------------------|

> | Event Title         | Git & GitHub through GitKraken - from Zero to Hero!       |

Reviewer: ```suggestion
| Event Title         | Introduction to Conda for (Data) Scientists         |
```

| Organiser           | RSE Sheffield                                             |
| Type                | Class, Training, or Workshop                              |
| Category            | Science & Technology                                      |

> | Tags                | git, github, gitkraken                                    |

Reviewer: ```suggestion
| Tags                | Conda, virtual environments, reproducibility                                    |
```

Author: Thanks for catching these and other oversights in this table, duly updated the table, apologies for skipping it.

| Location            | Online Event (unless otherwise)                           |
| Date and Time       | Single Event if running over one day, otherwise Recurring |
| Event Starts        | Start Date/Time (always Single Event), start is usually 09:30 |

> | Event Ends          | End Date/Time if one day finish is usually 16:30, if two 13:00 the following day.  |

Reviewer: ```suggestion
| Event Ends          | End Date/Time finish is usually 13:30.  |
```

### Description

> How can we undertake research work in a consistent and reproducible computing environment?
> This is an intermediate course, teaching the use of Conda virtual environments for reproducible research envuironments
> when using Python for research and data analysis. These skills are essential for collaborative research teams.
> This event is for University of Sheffield researchers and research students only. Please register with an @shef.ac.uk
> or @sheffield.ac.uk email address to ensure your ticket request is accepted.
>
> **Prerequisite skills**:
> - Basic shell commands (bash/zsh (Linux/MacOS); Powershell (Windows)).
> - Some Python programming experience.
>
> ### Learning Objectives
> - Understand what Conda is and why it is useful for managing packages and environments in research workflows.
> - How to create, activate and delete Conda environments.
> - How to install and manage packages in Conda environments.
> - What Conda channels and packages are and how to use them.
> - How to use pip in Conda environments.
> - How to share your Conda environment.
> - How to manage packages that have GPU dependencies.
>

> > This course will be run online.

Reviewer: Will it?

Author: I thought as it might be I'd leave the template as is, its currently how the Git course is configured and its down to the person scheduling to tweak this and the emails appropriately (could have two sets of templates but felt like overkill).

Author: Removed, its simple to do the reverse and add a comment that it will be online when we do.
